audio-worklet/bitcrusher-test.html

```diff
@@ -89,38 +100,42 @@
               // error occurs because any sample passing through bitcrusher will 
               // undergo manipulation in Math.round() and will therefore be 
               // represented by a new floating point number that differs
-              // slightly from the original.In tested samples and at a sample 
-              // rate of 44,1k, the maximum observed error is 
-              // 6.183981895446777e-7.
-              const permittedSampleError = 6.1840e-7;
+              // slightly from the original. In tested samples and at a sample 
+              // rate of 4096, the maximum observed error is 
+              // 2.9802322387695312e-8.
+              const permittedSampleError = 2.98025e-8;
               let maxError = 0;
+
               for (let i = 0; i < bitcrusherOutput.length; i++) {
-                let diff = Math.abs(bitcrusherOutput[i] - originalOutput);
+                const diff = Math.abs(bitcrusherOutput[i] - originalOutput[i]);
```

Comment:
Yes, the old code was stuck in a long loop. In other words, when you subtract an array from an element, no error is given and it it instead returns NaN eventually at least on small inputs from what I can tell. In this case, it must have done tons of computation trying to subtract the original output from the element and tricked me into thinking it had completed the tests.

I of course shouldn't have pushed the code in this state but in my defense it was late Friday night. This is an argument for having print statements when tests pass :). I am using Audit now for the noise gate tests, which is amazing, and can change these tests to support audit.